presto-bigquery/pom.xml

```diff
@@ -68,12 +68,6 @@
                 <version>0.22.2</version>
             </dependency>
 
-            <dependency>
-                <groupId>commons-codec</groupId>
-                <artifactId>commons-codec</artifactId>
-                <version>1.13</version>
```

Comment:
Why is this getting removed?

Reply:
This was in dependency management - since the version exists in the root pom, I removed it so that it defaults to using the root version.

Reply:
But then dependency would still be required right? We can avoid giving a version if this is already included in the root along with the version.

Reply:
This wasn't under the dependencies but under dependency management - so this was just a version override and not pulling in a dependency.